backend/vxlan/vxlan_network_windows.go

```diff
@@ -109,7 +109,7 @@ func (nw *network) handleSubnetEvents(batch []subnet.Event) {
 		managementIp := event.Lease.Attrs.PublicIP.String()
 
 		networkPolicySettings := hcn.RemoteSubnetRoutePolicySetting{
-			IsolationId:                 4096,
+			IsolationId:                 vxlanAttrs.VNI,
```

Comment:
Thanks for the PR! This is one of those lingering tasks we hadn't gotten around to.

Can you just add a check to make sure the VNI is 4096 or greater? That's a windows restriction :) Otherwise lgtm